### DISCUSSION TITLE
Welcome to course-project-npcs-ratemyuoft-project Discussions!

### DISCUSSION THREAD

**Discussion by IamMichael23** [CSC207-2022F-UofT/course-project-npcs-ratemyuoft-project#17] [Announcements]
Slides

**Comment by IamMichael23** [comment]
We need to start merging and getting the slides done.

**Comment by johnchakkour** [comment]
Yeah I've started work on the presentation. Input boundary, controller, presenter, output boundary, and of course the interactor will be crucially explained in detail. As discussed, we should make sure our code does not violate the Liskov Responsibility Principle.

**Comment by IamMichael23** [comment]
I think we need to add  GitHub Features, SOLID, Clean Architecture, Design Patterns on the slides

**Reply by ElangBogopa** [reply]
Let me know if you need help integrating with Leave Review
You can add things to the review class if you need to but please dont delete anything

**Comment by myk0laUA** [comment]
In case we will have a problem with merging view course, i recommend @PurpleChina23 either recording a demo of functionality of his branch, or work together with @KenanBogopa to make leave review also display all the reviews and add comments to them. As we don't have course pages yet, but we can leave reviews for courses.

**Comment by myk0laUA** [comment]
That would be great improvement in functionality. However, if we won't be able to get it on time, it won't be critical.

**Comment by myk0laUA** [comment]
In worst case scenario, I believe it can be done between the Milestone 5 submission deadline and final exam. I think, either professor or TA would take it into consideration, as we are lacking 1 team member and we had to redistribute the workload on others. @Shawndavison26 and @PurpleChina23 I believe it is a good idea to either email TA about that or make a private piazza post for prof to see that.
